beetsplug/lyrics.py

```diff
@@ -359,16 +359,10 @@ def __init__(self, config, log):
             'User-Agent': USER_AGENT,
         }
 
-    def lyrics_from_song_page(self, page_url):
-        # Gotta go regular html scraping... come on Genius.
-        self._log.debug(u'fetching lyrics from: {0}', page_url)
-        try:
-            page = requests.get(page_url)
-        except requests.RequestException as exc:
-            self._log.debug(u'Genius page request for {0} failed: {1}',
-                            page_url, exc)
-            return None
-        html = BeautifulSoup(page.text, "html.parser")
+    def scrape_lyrics_from_song_page(self, html):
+        """Scrape lyrics from a given genius.com song url"""
```

Comment:
Seems like a good idea, but this docstring is not quite accurate because there is no URL involved here—just the HTML source.

Reply:
Yep, my mistake. I wrote that docstring before I outsourced the html fetching from this method.